Refit/RestMethodInfo.cs

```diff
@@ -11,6 +11,19 @@
 
 namespace Refit
 {
+    public struct HeaderInfo
+    {
+        public bool IsHeaderCollection { get; }
```

Comment:
As per above, I'm not sure why this is required. There's already an attribute on the parameter, that should be enough?

Reply:
Ditto